modules/migration-using-must-gather.adoc

```diff
@@ -19,13 +27,33 @@ The `openshift-migration-must-gather-rhel8` image for {mtc-short} collects migra
 ----
 $ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v{mtc-version}
 ----
++
+The data is saved in a local `must-gather` directory.
+
+.Viewing metrics data with Prometheus
 
-. Remove authentication keys and other sensitive information.
-. Create an archive file containing the contents of the `must-gather` data directory:
+. Run the `oc adm must-gather` command with the `gather_metrics_dump` script:
 +
 [source,terminal]
 ----
-$ tar cvaf must-gather.tar.gz must-gather.local.<uid>/
+$ oc adm must-gather --image={must-gather} -- /usr/bin/gather_metrics_dump
```

Comment:
You must check the preview build as well. You will see that this attribute is not rendering. It does not exist in OCP docs. You must use the correct must-gather image name. OCP docs is a different repo from MTV, so you can't assume that they contain the same attributes.